spec/requests/providers/bank_statements_spec.rb

```diff
@@ -332,29 +332,11 @@
           end
         end
 
-        context "when HMRC response status is applicant_not_employed, " \
-                "but application is not using bank upload journey" do
+        context "when HMRC response status is applicant_not_employed" do
           before do
             allow(HMRC::StatusAnalyzer).to receive(:call).and_return :applicant_not_employed
           end
 
-          it "redirects to identify types of income page" do
```

Comment:
did you ever discover why this might have been necessary at some point, but 👍 regardless